docs/release-notes.md

```diff
@@ -3,6 +3,32 @@
 - Table of Contents
 {:toc}
 
+## Version 1.3.0
+
+The 1.3.0 release of MCUboot brings in many fixes and updates.  There
+are no breaking changes in functionality.  Many of the changes are
+refactorings that will make the code easier to maintain going forward.
+In addition, support has been added for encrypted images.  See [the
+docs](encrypted_images.md) for more information.
+
+### About this release
+
+- Modernize the Zephyr build scripts.
+- Add a `ptest` utility to help run the simulator in different
+  configurations.
+- Migrate the simulator to Rust 2018 edition.  The sim now requires at
+  least Rust 1.32 to build.
+- Simulator cleanups.  The simulator code is now built the same way
+  for every configuration, and queries the MCUboot code for how it was
+  compiled.
+- Abstract logging in MCUboot.  This was needed to support the new
+  logging system used in Zephyr.
+- Add multiple flash support.  Allows slot1/scratch to be stored in an
+  external flash device.
+- Add support for encrypted images.
```

Comment:
Please mention `docs/encrypted_images.md`

Reply:
I did in the textual description above, but I can duplicate that in the bullet let as well.